rebar.lock

```diff
@@ -92,7 +92,7 @@
  {<<"hackney">>,{pkg,<<"hackney">>,<<"1.15.2">>},0},
  {<<"helium_proto">>,
   {git,"https://github.com/helium/proto.git",
-       {ref,"648d7f73b9267fb4242427b93405db5b9966796e"}},
+       {ref,"45a0edda39eefc9b36e98c3a1019819e05321efc"}},
```

Comment:
That hash is currently 5 commits behind HEAD.  Was this intentional?

Reply:
yes